CCIP rate limits enhancement (#1463)
smartcontractkit/documentation: src/content/ccip/architecture.mdx

### Token pool rate limit

For each supported token on each individual [lane](/ccip/concepts#lane), the token pool rate limit manages the total number of tokens that can be transferred within a given time. This limit is independent of the USD value of the token.

For example, the maximum capacity of the LINK token pool on the Ethereum → Optimism lane is 100 LINK, and the refill rate is 1 LINK per second. If 100 LINK is transferred on that lane, the entire capacity would be consumed. After that, if a user wants to send 10 LINK, they must wait at least 10 seconds.

### Aggregate rate limit

Each [lane](/ccip/concepts#lane) also has an aggregate rate limit that limits the overall USD value that can be transferred on that lane across all supported tokens. To improve security, the aggregate rate limit for any given lane is always lower than the sum of all individual token pools rate limits for that lane.

> For example, suppose the Ethereum → Optimism lane has a maximum capacity of 100,000 USD, a refill rate of 100 USD per second, and several token transfers with a total value of 60,000 USD have been executed. In that case, the available capacity is 40,000 USD. So, if a user intends to transfer tokens equating to 50,000 USD, they would have to wait at least 100 seconds.

Reviewer: ```suggestion
Consider an example where a lane has a maximum capacity of 100,000 USD, a refill rate of 100 USD per second, and several token transfers with a total value of 60,000 USD have been executed. In that example, the available capacity is 40,000 USD. If a user intends to transfer tokens equating to 50,000 USD, they must wait at least 100 seconds.
```
